tox.ini

```diff
@@ -1,96 +1,94 @@
 [tox]
-envlist =
-        py{35}-django{111,20,21,22}-drf{39,310,latest},
-        docs,
-        quality,
-        version_check,
-        pii_check,
-        translations
+envlist = 
+	py{35,38}-django{22,30}-drf{39,310,latest},
+	docs,
+	quality,
+	version_check,
+	pii_check,
+	translations
 
 [testenv]
-deps =
-    django111: Django>=1.11,<2.0
-    django20: Django>=2.0,<2.1
-    django21: Django>=2.1,<2.2
-    django22: Django>=2.2,<2.3
-    drf39: djangorestframework<3.10.0
-    drf310: djangorestframework<3.11.0
-    drflatest: djangorestframework
-    -rrequirements/test.txt
-commands =
-    python -Wd -m pytest {posargs:-n 3}
+deps = 
+	django22: Django>=2.2,<2.3
```

Comment:
Why did the whitespace change here?  I thought we'd been using 4-space indentation pretty consistently in tox.ini files.  Making it hard to scan and see what actually changed.

Reply:
script replaced 4-spaces indentation with tab, should i revert whitespace changes?

Reply:
Yes, we definitely don't want tabs in these files; please update the script to use 4 space indentation instead.

Reply:
Correction: we didn’t add a leading tab, we should be able to update the constants DJANGO_22_DEPEDENCY and
DJANGO_30_DEPENDENCY to add leading spaces without making any other changes.

Reply:
As discussed in the group, the script does not make any change here. You can fix this by converting tabs to spaces in the file. In PyCharm, you can do this by selecting **edit->ConvertIndents->ToSpaces**.